docs/nginx.md

```diff
@@ -0,0 +1,42 @@
+# Nginx Receiver
+
+The `nginx` receiver can fetch stats from a NGINX instance using the mod_status endpoint.
+
+
+## Prerequisites
+
+[ngx_http_stub_status_module](http://nginx.org/en/docs/http/ngx_http_stub_status_module.html) must be enabled on the NGINX instance. This may require editing the NGINX configuration.
```

Comment:
Can we expand a bit on the details for what exactly users need to do. Some reference from the legacy monitoring agent doc: https://cloud.google.com/monitoring/agent/plugins/nginx#prerequisites

Reply:
I've expanded quite a bit, borrowing from the legacy docs.

@jsirianni, would you mind double checking my instructions?